package.json

```diff
@@ -32,8 +32,8 @@
     "prepackage": "npm run locales && npm run lint && svgo data/img",
     "prepush": "npm run package",
     "dev": "npm run locales && npm run watch-script & npm run watch & http-server -c-1",
```

Comment:
As far as I know, `http-server` is a package, not a ./bin/ script, see [package.json:54](https://github.com/meandavejustice/min-vid/blob/327820920aeb1e387e383c8372fe3fd2c62e2157/package.json#L54).